SaintBoolies/ClientApp/src/components/Sidebar.js

```diff
@@ -0,0 +1,55 @@
+import React,{ useState } from "react";
+import NoteGroup from "./NoteGroup";
+import NoteItself from "./NoteItself"
+
+const Sidebar = () => {
+  var check = true;
+
+  function toggle() {
+    var s1 = document.getElementById("s1");
+    var s2 = document.getElementById("s2");
```

Comment:
this in not the code which can be used in React. Please, replace it with react code approach.